src/components/ShareStructure/ShareStructure.vue

```diff
@@ -476,6 +488,10 @@ export default class ShareStructure extends Mixins(ShareMixin) {
   @Prop({ default: false })
   readonly invalidSection!: boolean
 
+  /** Verification the Share Structure contains the minimum required Share Classes. */
+  @Prop({ default: false })
+  readonly invalidMinimumShareClass!: boolean
```

Comment:
I think this should be generalized so it can be reused for possible future additional checks.

Also, this component doesn't really "know" what the invalid condition is, nor does it (or should it) care, which is another reason to rename this.

Reply:
🤔 I thought about that too but the styling effect it has on the component is specific. 
ie the invalidSection and invalidMinimum.. have different style outcomes in the UI. 
So as long as our designs call for specific messages or styling, i think we need to be specific no?

Reply:
You're right about the message, but the styling is generic right? I guess you could pass in the message too.

Reply:
Not exactly generic. 
The css is for sure but where it applies is not, ie different scenarios call for different places to apply the styling. 
We could pass in the message, but it's just another prop to handle and until we have a requirement for multiple messages i think it's ok to have a single message maybe?

Reply:
OK, leave it as is. Yes, we can review this if we need another message. No problem!

Reply:
Sounds good, thanks. Good discussion though, as always :)